<launch>

<node
pkg = "rosbag"
type = "play"
name = "play"
args = "/LidarCalib/lidarup.bag"
/>

<!--output = "screen"-->
<!--其中-l = -loop，即循环播放bag包-->

<node
pkg="lidar_calib"
type="lidar_calib"
name="calib_node"
output = "screen"
>
<param name="de_rz" value= "-17" type="double"/>
</node>

<!--这里的de_rz的单位是degree度数，在程序中转rad单位-->

<node
pkg = "rosbag"
type = "record"
name = "transbag"
args = "-O /home/wyc/mygit/LidarCalib/lidarupmid.bag /lidarcalib"
/>

<node
pkg = "rviz"
type = "rviz"
name = "rviz"
args = "-d /home/wyc/mygit/LidarCalib/calib.rviz"
required = "true"
/>

</launch>
